nailgun/entities.py

```diff
@@ -2195,7 +2196,11 @@ def read(self, entity=None, attrs=None, ignore=('operatingsystem',)):
 
 
 class Organization(
-        Entity, EntityCreateMixin, EntityDeleteMixin, EntityReadMixin):
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin,
+        EntityUpdateMixin):
```

Comment:
I'm going to submit at least one more PR before releasing 0.17.0. I'm going to update many of the tests in Robottelo to use the `update*` methods, add `EntityUpdateMixin` to many of the entities, and post the test results.